client/marketing/components/slider/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { Component, createRef } from '@wordpress/element';
+import classnames from 'classnames';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss'
+
+class Slider extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			height: null,
+		};
+		this.container = createRef();
+		this.onExited = this.onExited.bind( this );
+		this.onEnter = this.onEnter.bind( this );
+		this.updateSliderHeight = this.updateSliderHeight.bind( this );
+	}
+
+	/**
+	 * Update the slider height on Resize
+	 */
+	componentDidMount() {
+		// Update the slider height on Resize
+		window.addEventListener( 'resize', this.updateSliderHeight );
+	}
+
+	componentWillUnmount() {
+		window.removeEventListener( 'resize', this.updateSliderHeight )
+	}
+
+	updateSliderHeight() {
+		const slide = this.container.current.getElementsByClassName( 'woocommerce-marketing-slider__slide' )[ 0 ];
+		this.setState( { height: slide.clientHeight } );
+	}
+
+	onExited() {
+		const { onExited } = this.props;
+		if ( onExited ) {
+			onExited( this.container.current );
+		}
+	}
```

Comment:
the naming is here a bit confusing and might even trigger a [`no-shadow`](https://eslint.org/docs/rules/no-shadow) eslint error

Reply:
@jconroy looks like this is copy paste off of the AnimationSlider component? We could remove it since we're not actually using it. What do you think?

Reply:
Yep lets do it